packages/datatrak-web/src/api/queries/useTasks.ts

```diff
@@ -25,6 +30,36 @@ export interface UseTasksQueryParams {
   >[];
 }
 
+interface UseTasksLocalContext extends LocalContext {
+  projectId?: string;
+  allAssignees?: boolean;
+  includeCancelled?: boolean;
+  includeCompleted?: boolean;
+  pageSize?: number;
+  page?: number;
+  filters?: Filter[];
+  sortBy?: SortingRule<
+    // HACK: This should be derived from `DatatrakWebTasksRequest`
+    Record<'assignee_name' | 'due_date' | 'entity.name' | 'survey.name', unknown>
+  >[];
+}
+
+const getRemoteTasks = async ({
+  pageSize,
+  page,
+  filters,
+  sortBy,
+}: UseTasksLocalContext) => {
+  return get('tasks', {
+    params: {
+      pageSize,
+      page,
+      filters,
+      sort: sortBy?.map(({ id, desc }) => `${id} ${desc ? 'DESC' : 'ASC'}`) ?? [],
```

Comment:
`sort: undefined` is fine, right?

```suggestion
      sort: sortBy?.map(({ id, desc }) => `${id} ${desc ? 'DESC' : 'ASC'}`),
```